addons/project/models/project.py

```diff
@@ -1817,6 +1817,14 @@ def stage_find(self, section_id, domain=[], order='sequence, id'):
         # perform search, return the first found
         return self.env['project.task.type'].search(search_domain, order=order, limit=1).id
 
+    @api.model
+    def get_sellable_line_domain(self, domain):
+        sellable_domain = str(self.env['sale.order.line']._sellable_lines_domain())
+        if not domain:
+            return sellable_domain
+        domain = domain.replace('\r\n', '').replace('\n', '').strip()[1:-1].strip(", ")
+        return f"['&', {domain}, {sellable_domain[1:-1]}]"
```

Comment:
this method should be in sale_project module since `sale.order.line` does not exist if we just install project module.